whois.go

```diff
@@ -11,6 +12,8 @@ const (
 	ianaWHOISServerAddress = "whois.iana.org:43"
 )
 
+var domainExtensionsWithoutGracePeriod = []string{"at","be","ch","co.at","com.br","or.at","de","fr","me","mx","nl"}
```

Comment:
Could we rename `domainExtensionsWithoutGracePeriod` and `isDomainExtensionWithoutGracePeriod()` to something like `tldWithoutExpirationDate` and `doesTLDHaveExpirationDate`?

Reply:
Yes, good suggestion. Done!